app/src/main/res/values/strings.xml

```diff
@@ -935,6 +935,7 @@
     <string name="anitrend_sign_in_browser">Google Chrome Recommended</string>
 
     <!-- Privacy Settings -->
+    <string name="pref_key_privacy" translatable="false">privacy</string>
```

Comment:
You may get a merge conflict on 868d80ce I moved all preference keys into `app/src/main/res/values/preference_keys.xml` sorry about that